author.tex

```diff
@@ -0,0 +1,389 @@
+
+\author[0000-0003-0872-7098]{A. M. Price-Whelan}
+\affil{Department of Astrophysical Sciences, Princeton University, Princeton, NJ 08544, USA}
```

Comment:
Might be overkill, but I wonder if we should define macros for each affiliation to just ensure that places that appear multiple times have the same address?